config/config.go

```diff
@@ -840,6 +840,11 @@ type MempoolConfig struct {
 	// the $CMTHOME env variable or --home cmd flag rather than overriding this
 	// struct field.
 	RootDir string `mapstructure:"home"`
+	// Mempool reactor, valid options:
+	// - "cat": reactor with push-pull gossip protocol
+	// - "" (default): reactor with flooding protocol
+	Reactor string `mapstructure:"reactor"`
```

Comment:
I would not call this parameter `reactor` - that's a concept that's internal to the CometBFT architecture that shouldn't be leaked to operators (such leakage is bad UX).

Operators would probably understand this better as `type` (as in the user story: "as an operator, I can select which type of mempool to run").

Reply:
I agree. I renamed it though to `gossip_protocol`, which I think it's more precise.